ci/config_propel.php

```diff
@@ -0,0 +1,92 @@
+use Spryker\Zed\Propel\PropelConfig;
+use Spryker\Zed\PropelOrm\Business\Builder\ExtensionObjectBuilder;
+use Spryker\Zed\PropelOrm\Business\Builder\ExtensionQueryBuilder;
+use Spryker\Zed\PropelOrm\Business\Builder\ObjectBuilder;
+use Spryker\Zed\PropelOrm\Business\Builder\QueryBuilder;
+
+$placeholder = '%s:host=%s;port=%d;dbname=%s';
+
+$dsn = sprintf(
+    $placeholder,
+    $config[PropelConstants::ZED_DB_ENGINE],
+    $config[PropelConstants::ZED_DB_HOST],
+    $config[PropelConstants::ZED_DB_PORT],
+    $config[PropelConstants::ZED_DB_DATABASE]
+);
+
+$slaves = [];
+foreach ($config[PropelConstants::ZED_DB_REPLICAS] ?? [] as $slaveData) {
+    $slaves[] = [
+        'dsn' => sprintf(
+            $placeholder,
+            $config[PropelConstants::ZED_DB_ENGINE],
+            $slaveData[PropelConstants::ZED_DB_HOST],
+            $slaveData[PropelConstants::ZED_DB_PORT],
+            $config[PropelConstants::ZED_DB_DATABASE]
+        ),
+        'user' => $config[PropelConstants::ZED_DB_USERNAME],
+        'password' => $config[PropelConstants::ZED_DB_PASSWORD],
+    ];
+}
+
+$connections = [
+    'pgsql' => [
+        'adapter' => PropelConfig::DB_ENGINE_PGSQL,
+        'dsn' => $dsn,
+        'user' => $config[PropelConstants::ZED_DB_USERNAME],
+        'password' => $config[PropelConstants::ZED_DB_PASSWORD],
+        'slaves' => $slaves,
+        'settings' => [],
+    ],
+    'mysql' => [
+        'adapter' => PropelConfig::DB_ENGINE_MYSQL,
+        'dsn' => $dsn,
+        'user' => $config[PropelConstants::ZED_DB_USERNAME],
+        'password' => $config[PropelConstants::ZED_DB_PASSWORD],
+        'slaves' => $slaves,
+        'settings' => [
+            'charset' => 'utf8',
+            'queries' => [
+                'utf8' => 'SET NAMES utf8 COLLATE utf8_unicode_ci, COLLATION_CONNECTION = utf8_unicode_ci, COLLATION_DATABASE = utf8_unicode_ci, COLLATION_SERVER = utf8_unicode_ci',
+            ],
+        ],
+    ],
+];
+
+$config[PropelConstants::PROPEL] = [
+    'database' => [
+        'connections' => [],
+    ],
+    'runtime' => [
+        'defaultConnection' => 'default',
+        'connections' => ['default', 'zed'],
+    ],
+    'generator' => [
+        'defaultConnection' => 'default',
+        'connections' => ['default', 'zed'],
+        'objectModel' => [
+            'defaultKeyType' => 'fieldName',
+            'builders' => [
+                // If you need full entity logging on Create/Update/Delete, then switch to
+                // Spryker\Zed\PropelOrm\Business\Builder\ObjectBuilderWithLogger instead.
+                'object' => ObjectBuilder::class,
+                'objectstub' => ExtensionObjectBuilder::class,
+                'query' => QueryBuilder::class,
+                'querystub' => ExtensionQueryBuilder::class,
+            ],
+        ],
+    ],
+    'paths' => [
+        'phpDir' => APPLICATION_ROOT_DIR,
+        'sqlDir' => APPLICATION_ROOT_DIR . 'src/Orm/Propel/Sql',
```

Comment:
This and all following should always have trailing slash

Reply:
Fixed.